posnums.c

```diff
@@ -0,0 +1,95 @@
+/*Ashank Kumar find average, if numbers are prime and if they are relatively prime*/
```

Comment:
Nice job for the most part. You don't handle every single case, however. Specifically, when a user inputs a prime number as the second number, it doesn't output whether or not it's a prime number.